File: Basic_Algorithms/002_check_even/check_even.cpp
#include <iostream>

bool even(int);

int main(){
  int x = 10;

  std::cout << x << " is even : " << std::boolalpha << even(x) << std::endl;
}

bool even(int n){
  if(n % 2 == 0) return true;
  else return false;
}

File: Basic_Algorithms/002_check_even/check_even.java
class Check_Sum{
  public static void main(String[] args){
    int x = 2;

    System.out.println(x + " is even : " + isEven(x));
  }

  static boolean isEven(int n){
    if(n % 2 == 0) return true;
    return false;
  }
}
